src/OrbitQt/CMakeLists.txt

```diff
@@ -115,12 +120,15 @@ target_sources(OrbitQtTests
                 AnnotatingSourceCodeDialogTest.cpp
                 CallTreeViewItemModelTest.cpp
                 StatusListenerImplTest.cpp
-                TutorialOverlayTest.cpp)
+                TutorialOverlayTest.cpp
+                TrackTypeItemModelTest.cpp
+                ../OrbitGl/TrackManagerTest.cpp)
```

Comment:
It shouldn't be done like this. I assume that this is to get the definition of `GenerateTestCaptureData`, which further supports that that method shouldn't be defined in that file.

Reply:
I'm now building `TrackTestData.cpp`, but I'm assuming you are questioning the inclusion of a file from another module in general, not only the naming?

TBH I'm not sure how we want to handle test data that we share across modules. Maybe shared test data should live in a separate module?

Reply:
> I'm assuming you are questioning the inclusion of a file from another module in general, not only the naming

Yes, let's please not start doing that, otherwise we should just merge `OrbitGl` and `OrbitQt` together.

A simple way would be to have `TrackTestData` not in `OrbitGlTests` but in `OrbitGl`, which both `OrbitGlTests` and `OrbitQtTests` depend on.

Reply:
Right... I didn't do that initially due to the dependency on gtest, but of course that's no longer an issue, so I can just move them to OrbitGl. PTAL.